Automattic/jetpack
# VideoPress: Fix focus style on ClipboardButtonInput component

Fixes #26345

#### Changes proposed in this Pull Request:

* Removes the box-shadow of the input element and adds a focus style to the whole component in line with the other focusable components

#### Other information:


#### Jetpack product discussion


N/A

#### Does this pull request change what data or activity we track or use?


No.

#### Testing instructions:






* Go to the VideoPress dashboard
* Click `Edit video details` on a video
* Focus the `Link to video` input
* Check that there is no blue box-shadow around the input element
* Check that there is a black box-shadow around the entire component

![2022-10-17_11-43-39](https://user-images.githubusercontent.com/8486249/196208925-9e19fd23-5941-4681-9c2b-acfedfa91c5c.png)

https://user-images.githubusercontent.com/8486249/196208970-c86a5a44-4be1-4cca-8d70-c996e188c1f0.mp4
